### ISSUE TITLE
IE10 Cross Domain Issue

### ISSUE THREAD

**Issue by jeffchannell** [recurly/recurly-js#95]
I'm having trouble with PayPal and Recurly playing nice in IE10.

We are currently testing this issue on an account that does not have PayPal enabled, but the production account has the same issue. During testing we get back a script from Recurly that contains this comment (which means it's a known issue of sorts):

```
      // Recurly.js will close this window if postMessage succeeds.
      // IE10 fails cross-origin postMessage silently, so to handle this,
      // we try using window.name after 1 second which will only
      // execute if this window is still open.
```

This interval on the client is supposed to run until the popup window changes the parent window's name:

```
    var interval = setInterval(function() {
      var decoded = decodeURIComponent(window.name)
        , match = decoded.match(/recurly_result=(.*)[&$]?/)
        , result = match && $.parseJSON(match[1]);

      if(result) {
        finish(result);
        window.name = originalWindowName;
      }

    }, 1000);
```

However, this interval runs continuously and never finishes. Setting a breakpoint at the beginning of this and watching window.name in IE10 developer tools confirmed that the child window NEVER sets the parent window's name because it is not allowed by the browser.

The best part is that while investigating the issue (using breakpoints) I could occasionally trigger something in IE that would cause the code to work correctly, however I'm not entirely sure how to replicate and resetting the browser (via Internet Options) and rebooting makes this happen once again.

tl;dr - In my case Recurly is not playing nice with default Win7/IE10 cross domain settings.
